### PR TITLE
streaming_threshold not None check. Also dropping `loop` parameter from aiobotocore tests.

### PR DESCRIPTION
**Pull request by srprash**
*Issue #, if available:* [201](https://github.com/aws/aws-xray-sdk-python/issues/201)

*Description of changes:*
While setting the streaming_threshold in `xray_recorder.configure()`, if the value is 0, the condition evaluates to false if not checked for None.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -29,9 +29,13 @@ deps =
     py{27}: enum34
 
     # Python3.5+ only deps
-    py{35,36,37,38}: aiohttp >= 3.0.0
+    py{35,36,37,38}: aiohttp >= 3.3.1
```
Comment (bhautikpip) [inline tox.ini]: Have they deprecated support for aiohttp 3.0.0 ?

Comment (srprash) [reply tox.ini]: I don't think so. But the aiobotocore setup file requires aiohttp >= 3.3.1 so I made that change too. I'm not sure if it has any implications.

File: tox.ini [hunk 2]
```diff
@@ -29,9 +29,13 @@ deps =
...
     py{35,36,37,38}: pytest-aiohttp
-    py{35,36,37,38}: aiobotocore >= 0.10.0
+    py{35,36,37,38}: botocore >= 1.15.3,<1.15.16
+    # aiobotocore 0.12.0 is not available for python 3.5
+    py{35}: aiobotocore >= 0.10.0
```
Comment (bhautikpip) [inline tox.ini]: I think it will look for latest version so you can add this case with other cases. py{35,36,37,38} >= 0.12.0

File: tox.ini [hunk 3]
```diff
@@ -29,9 +29,13 @@ deps =
     py{27}: enum34
 
     # Python3.5+ only deps
-    py{35,36,37,38}: aiohttp >= 3.0.0
+    py{35,36,37,38}: aiohttp >= 3.3.1
     py{35,36,37,38}: pytest-aiohttp
-    py{35,36,37,38}: aiobotocore >= 0.10.0
+    py{35,36,37,38}: botocore >= 1.15.3,<1.15.16
```
Comment (bhautikpip) [inline tox.ini]: <1.15.16 means we will have to keep updating this version number in the case of future updates with botocore. I think botocore >= 1.15.3 should be good.

File: tox.ini (hunk 2 above)
Comment (srprash) [reply tox.ini]: actually we may not even need this case at all. having py{35,36,37,38} >= 0.10.0 should pull in whatever latest version is supported for each runtime.

File: tox.ini (hunk 3 above)
Comment (srprash) [reply tox.ini]: You're right. Probably don't need this one as well.

File: tox.ini (hunk 2 above)
Comment (bhautikpip) [reply tox.ini]: Yeah. That's what I am suggesting. we can keep 0.10.0

File: tox.ini (hunk 1 above)
Comment (bhautikpip) [reply tox.ini]: Okay!

File: tox.ini [hunk 4]
```diff
@@ -98,4 +98,4 @@ commands =
 
 [flake8]
 max-line-length=120
-exclude=tests
+exclude=tests
```
Comment (bhautikpip) [inline tox.ini]: Can you add a new line here?

**Pull request merged**